test/timed-stream.ts

```diff
@@ -24,13 +24,23 @@ function* numberGenerator(n: number) {
   }
 }
 
+class UserStream extends TimedStream {
+  constructor() {
+    super({
+      transformHook(event, _encoding, callback) {
+        callback(null, event);
+      },
+    });
+  }
+}
+
```

Comment:
This is now necessary because we have to tell the TimedStream to use the callback in the transformHook now.